cg/cli/workflow/nextflow/options.py

```diff
@@ -0,0 +1,49 @@
+"""CLI support for nextflow pipelines."""
+
+import click
+
+OPTION_LOG = click.option(
+    "-log",
+    type=click.Path(),
+    help="Set nextflow log file path",
+)
+OPTION_WORKDIR = click.option(
+    "-work-dir",
+    type=click.Path(),
+    help="Directory where intermediate result files are stored",
+)
```

Comment:
As far as I understand these are constructed by default if not passed. I think these options are currently here to add flexibility and be able to easily change the default values. But if I understand this will probably be removed and replaced by a config at some point